### PR TITLE
Update meta-quest.md

### PR DESCRIPTION
**Pull request by slaygirlz**
looks like the quest 3s wasn't added 👍

### CODE REVIEW

File: docs/meta-quest/meta-quest.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Home Assistant has started to offer the [minimal flavor](/core/android-flavors.m
 | Meta Quest | Yes |
 | Meta Quest 2 | Yes |
 | Meta Quest Pro | Yes |
-| Meta Quest 3 | Yes |
+| Meta Quest 3/3s | Yes |
```
Comment (jpelgrom) [inline docs/meta-quest/meta-quest.md:21]: Let's match Meta's branding

```suggestion
| Meta Quest 3/3S | Yes |
```

**Review by jpelgrom — CHANGES_REQUESTED** [review]
See my previous comment

**Comment by slaygirlz** [comment]
sorry! I thought that maintainers can override pulls but here is the updated one

**Comment by jpelgrom** [comment]
> sorry! I thought that maintainers can override pulls but here is the updated one

Yes, but you could've disagreed and it's not urgent :)

**Review by jpelgrom — APPROVED** [review]
Thanks!

**Pull request merged**